wave_vision/include/wave/vision/tracker/tracker.hpp

```diff
@@ -0,0 +1,171 @@
+/**
+ * @file
+ * Brute force matcher implementation, derived from descriptor matcher base.
+ * @ingroup vision
+ */
+#ifndef WAVE_VISION_TRACKER_HPP
+#define WAVE_VISION_TRACKER_HPP
+
+#include <chrono>
+#include <string>
+#include <vector>
+
+#include <opencv2/opencv.hpp>
+
+#include "wave/containers/landmark_measurement.hpp"
+#include "wave/containers/landmark_measurement_container.hpp"
+#include "wave/utils/utils.hpp"
+#include "wave/vision/utils.hpp"
+
+namespace wave {
+/** @addtogroup vision
+ *  @{ */
+
+class FeatureTrack {
```

Comment:
So after talking with @chutsu offline, I think we'd like to keep the FeatureTrack class. This is primarily because of the first/last_image parameters. This may tie in to the other comment regarding times/image counts for the measurement container...we should discuss this offline @leokoppel.